project/data_analysis/python/planck/README.rst

```diff
@@ -0,0 +1,40 @@
+*******************
+Planck maps pre-processing
+*******************
+
+This directory holds tools to pre-process Planck NPIPE maps used to perform calibration and consistency tests.
+
+The first step is the projection Planck NPIPE maps publicly available at ``NERSC`` into a CAR pixellization. To project 100,143,217 and 353 GHz maps (8 maps), one should run
+
+.. code:: shell
+
+    salloc -N 1 -C cpu -q interactive -t 02:00:00
+    srun -n 8 -c 32 --cpu_bind=cores python project_planck_npipe_maps.py planck_pre_processing.dict
+
+Once projected we might want to use them to compute power spectra or to get source subtracted maps. It requires to get data products such as beams which can be written to disk with
+
+.. code:: shell
+
+    python get_planck_npipe_beams.py
+
+To perform source subtraction using `dory`, one needs to get a single frequency source catalog by running ``reformat_source_catalog.py`` that produces three catalogs from the ACT DR6 multi-frequency catalog
+
+.. code:: shell
+
+    python reformat_source_catalog.py
+
+The source subtraction can then be performed at ``NERSC`` using the following instructions
+
+.. code:: shell
+
+    salloc -N 1 -C cpu -q interactive -t 02:30:00
+    source run_npipe_src_subtraction.sh
```

Comment:
You do not need to `source` since there is an explicit shebang (`#!/usr/bin/env bash`) in the header of the file, so running `./run_npipe_src_subtraction.sh` is enough (given the file has been `chmod a+x`)